### PR TITLE
Update Orb to include inline scanner

### PR DESCRIPTION
**Pull request by aedifex**
@afiune @ipcrm Opening this PR to get the conversation started on updating the Orb to include functionality for the inline scanner. The problem can be broken up into a few discrete steps as follows.

1. Install lw-scanner in CircleCI environment

Providing a first class shell script to perform this is generally the way to go. Whereas the CLI has a first class installer, blessed by engineering, we are on the hook for maintaining our own variant for the CLI. Matt C. has done a great job leveraging prior art and porting it over to the inline scanner. @afiune does it make sense to keep that script in this repo?

2. Execute inline scan and parameterize cli arguments

The inline scanner has a primary use-case; executing scans against docker images, either from a registry or images present on the local filesystem. There are several flags and parameters available, e.g. persisting scan results as an HTML file as a "build artifact". Some of the configuration options are not applicable to the ephemeral nature of CI environments.

3. Testing

We can orchestrate a few testing scenarios where we perform scans against vulnerable images. Some of the additional options like returning non-zero exit codes are much more challenging because non-zero exit codes cause CircleCI to fail the pipeline. We can explore additional options.

Getting the right scanner args and parameters Orbifed shouldn't take _too_ long, aiming to have more progress shortly and hopefully we can close this out by Friday 11/20.

### CODE REVIEW

File: src/commands/install-lw-scanner.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+description: >
+  Install the Lacework lw-scanner
+steps:
+  - run:
+      name: Installing the lacework lw-scanner
+      command: |
+        curl https://raw.githubusercontent.com/ipcrm/circleci-orb-lacework/ally-713/src/scripts/install_lwscanner.sh | sudo bash
```
Comment (afiune) [inline src/commands/install-lw-scanner.yml]: can we merge this first into this repository, @ipcrm ?

Comment (ipcrm) [reply src/commands/install-lw-scanner.yml]: @aedifex Can you copy/add the script to this repo as part of this PR?  That was just where I had done the initial creation but not meant to be permanent.

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,6 +28,36 @@ jobs:
...
+    steps:
+      - checkout
+      - lacework/install-lw-scanner
+      - run:
+          name: Pull in vuln image for testing
+          command: |
+            docker pull chriscircleci/vuln_django:7f7305bea884b63578c28d4513a916e2247ec152
+      - lacework/inline-scanner-scan:
+          registry: ""
+          repository: ""
```
Comment (afiune) [inline .circleci/config.yml]: what are these two parameters for?

Comment (ipcrm) [reply .circleci/config.yml]: @aedifex to be removed

File: src/commands/install-lw-scanner.yml (hunk 1 above)
Comment (ipcrm) [reply src/commands/install-lw-scanner.yml]: @aedifex added this

File: src/commands/scanner-vuln-scan.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  non-os-packages:
+    description: Scan non-os packages
+    type: boolean
+    default: false
+steps:
+  - run:
+      name: Execute inline container scan
+      command: |
+        docker pull chriscircleci/vuln_django:7f7305bea884b63578c28d4513a916e2247ec152
+        lw-scanner evaluate chriscircleci/vuln_django
```
Comment (ipcrm) [inline src/commands/scanner-vuln-scan.yml]: @aedifex are these supposed to be static values here or should this be templates for the customer inputs?

File: .circleci/config.yml (hunk 2 above)
Comment (aedifex) [reply .circleci/config.yml]: Good catch - @afiune @ipcrm. It's been removed.

**Review by afiune — APPROVED** [review]
tenor-128120197

File: src/commands/install-inline-scanner.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+description: >
+  Install the Lacework lw-scanner
+steps:
+  - run:
+      name: Installing the lacework lw-scanner
+      command: |
+        curl https://raw.githubusercontent.com/lacework/circleci-orb-lacework/lw-scanner/scripts/install.sh | sudo bash
```
Comment (ipcrm) [inline src/commands/install-inline-scanner.yml:7]: @aedifex totally get this needs to be here; probably should raise a JIRA/TODO/whatever to swap this out after merged.

**Review by ipcrm — APPROVED** [review]

**Pull request merged**